cmd/kperf.go

```diff
@@ -0,0 +1,29 @@
+// Copyright © 2020 The Knative Authors
```

Comment:
I think our copyright header is (c), feels a bit off :)

Reply:
@vagababov Nice catch. Just want to double confirm, should I follow (C)
or withour C like kn clien did `Copyright 2020 The Knative Authors`
https://github.com/knative/client/blob/master/pkg/apis/client/register.go#L2

Reply:
I think without C, it's also what we are using for serving.